### ISSUE TITLE
Support for multiple computer / KVM setup

### ISSUE THREAD

**Issue by dfeinzimer** [1kc/razer-macos#452]
I've got two computers set up at my desk, a Mac and a PC. Both machines are on a KVM switch so I can switch my keyboard, mouse, and other peripherals between the two machines at the touch of a button. 

On the PC I'm running the official Razer software and on my Mac, this software. When I switch from Mac to PC the Razer software automatically picks up the connection and starts my last configured color mode. When I switch from PC to Mac, my keyboard goes completely dark and I first need to hit the `razer-macos` menubar icon then do -> `Refresh Device List` -> and then select my preferred lights. At that point, the lights will finally come on.

Basically, is it possible for `razer-macos` to monitor USB connections and when the keyboard is reconnected (via KVM in this instance) to auto refresh the device list and re-start my last used lights? Razer seems to do this perfectly fine in their official Windows software.

**Comment by sgulls** [comment]
+1 from me

**Comment by dfeinzimer** [comment]
Also see #206

**Comment by mgallag** [comment]
+1

It would be great to have different peripheral colors depending on whether I was controlling my Mac or PC (actually useful RGB!) 

Strangely, the mouse (Razer Lancehead TE) settings update automatically when switching to Mac but the keyboard (BlackWidow Chroma TE) persists whichever color the PC last set it.

Device emulation on the HID ports is not a factor since I have my mouse and keyboard connected via USB3 hub to the shared USB3 port on my KVM.

I tried working around the issue using state manager settings but there does not appear to be a "connect device" event to hook into.
